<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionaddActionParticipant"/>
    <addaction name="actionaddActionOrganizatie"/>
    <addaction name="actionaddActionCategorie"/>
    <addaction name="actionaddActionClasament"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionaddActionParticipant">
   <property name="text">
    <string>addActionParticipant</string>
   </property>
  </action>
  <action name="actionaddActionOrganizatie">
   <property name="text">
    <string>addActionOrganizatie</string>
   </property>
  </action>
  <action name="actionaddActionCategorie">
   <property name="text">
    <string>addActionCategorie</string>
   </property>
  </action>
  <action name="actionaddActionClasament">
   <property name="text">
    <string>addActionClasament</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
